pkgs/applications/graphics/krita/wrapper.nix

```diff
@@ -0,0 +1,26 @@
+{ callPackage
+, symlinkJoin
+, makeWrapper
+, krita-unwrapped
+, krita-plugin-gmic
+, plugins ? [
+    krita-plugin-gmic
+  ]
+}:
+
+symlinkJoin {
+  name = "krita-${krita-unwrapped.version}";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  paths = [ krita-unwrapped ] ++ plugins;
+
+  postBuild = ''
+    wrapProgram "$out/bin/krita" \
+      --set KRITA_PLUGIN_PATH "$out/lib/kritaplugins"
+  '';
+
+  meta = krita-unwrapped.meta // {
+    mainProgram = "krita";
```

Comment:
Why can't we inherit that, too?

Reply:
`krita-unwrapped` does not set `mainProgram`, I should add it there and just inherit it here.

Reply:
Added `mainProgram` in `krita-unwrapped` :+1: